frontend/ui/data/creoptions.lua

```diff
@@ -254,7 +254,7 @@ Note that your selected font size is not affected by this setting.]]),
                 more_options = true,
                 more_options_param = {
                   value_min = 70,
```

Comment:
I wouldn't mind lowering this to 50, too ;p.

Reply:
(I have a vague memory that Noto has insanely large line-gaps by default, and 70 was still pretty roomy. I may be misremembering :D).

Reply:
Okay, scratch that, I had tried Noto, and 70 is when ascenders and descenders start to meet, so, eh, fine by me at 70 ;).

Sorry for the noise.

Reply:
Done.

Reply:
(Can't hurt anyway, thanks!)

Reply:
Oh, github comments lags.
Well, still keeping the 50, might be useful with some crappy large line height fonts.